packages/framer-motion/src/render/dom/utils/unit-conversion.ts

```diff
@@ -277,6 +282,9 @@ const checkAndConvertChangedValueTypes = (
         // Reapply original values
         visualElement.syncRender()
 
+        // Restore scroll position
+        if (scrollY !== null) window.scrollTo({ top: scrollY })
```

Comment:
Is it possible that the scroll position also changed meanwhile, that we don't wanna restore the prev value?

Reply:
This is all synchronous, it's not batched like layout animations, much older code, but even so there couldn't be a change in between.